db/compaction/compaction_picker.h

```diff
@@ -78,7 +88,8 @@ class CompactionPicker {
       const CompactRangeOptions& compact_range_options,
       const InternalKey* begin, const InternalKey* end,
       InternalKey** compaction_end, bool* manual_conflict,
-      uint64_t max_file_num_to_ignore, const std::string& trim_ts);
+      uint64_t max_file_num_to_ignore, const std::string& trim_ts,
+      const SequenceNumber earliest_mem_seqno = kMaxSequenceNumber);
```

Comment:
How many callers would we have to update to avoid a default value? We should try to avoid introducing default values that are often incorrect. Keep in mind that this PR is fixing callers that relied on the incorrect `earliest_mem_seqno` default value in `FindIntraL0Compaction()`.

Reply:
Fixed - basically just  1-2 test files involved.